common/changes/office-ui-fabric-react/nav_2018-09-06-19-08.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "",
+      "packageName": "office-ui-fabric-react",
+      "type": "none"
```

Comment:
I don't think none is the right thing here. The documentation/example change is something we want to capture and note in the changelogs. It would also be something to trigger a new deployment of the website for (as opposed to say if we change a unit test).

Reply:
dito

Reply:
okay!